powershell/VstsTaskSdk/LoggingCommandFunctions.ps1

```diff
@@ -554,7 +560,7 @@ function Write-LogIssue {
         [switch]$AsOutput,
         [AllowNull()]
         [ValidateSet('CustomerScript', 'TaskInternal')]
-        [string]$IssueSource)
+        [string]$IssueSource = $IssueSources.TaskInternal)
```

Comment:
What if a non-builtin task task calls Write-LogIssue.  Will "IssueSource" be ignored?

Reply:
It won't be ignored, a user can see this property if the pipeline is executing with system diagnostic enabled (debug mode). However, it won't affect telemetry since ADO ignores issues for 3rd party tasks at all.